### PR TITLE
feat: get_chunks_info query method

### PR DESCRIPTION
**Pull request by 3cL1p5e7**
We need to know chunks info of any asset for developing streaming strategy on client

Linked [PR](https://github.com/dfinity/certified-assets/pull/24)

### CODE REVIEW

File: src/ic-certified-assets/src/lib.rs [hunk 1]
```diff
@@ -203,6 +203,18 @@ struct HttpRequest {
     body: ByteBuf,
 }
 
+#[derive(Clone, Debug, CandidType, Deserialize)]
+struct ChunkInfo {
+    chunk_id: ChunkId,
+    total_length: Nat,
```
Comment (mstein-dfinity) [inline src/ic-certified-assets/src/lib.rs:209]: I would rename this maybe to chunk_length

Comment (mstein-dfinity) [reply src/ic-certified-assets/src/lib.rs:209]: and maybe make it u64

Comment (3cL1p5e7) [reply src/ic-certified-assets/src/lib.rs]: @mstein-dfinity done :)

File: src/ic-certified-assets/src/lib.rs [hunk 2]
```diff
@@ -203,6 +203,18 @@ struct HttpRequest {
     body: ByteBuf,
 }
 
+#[derive(Clone, Debug, CandidType, Deserialize)]
+struct ChunkInfo {
```
[the remark this answers is not in the document]
Comment (3cL1p5e7) [reply src/ic-certified-assets/src/lib.rs:207]: @jplevyak 
yes, most likely it is better to make a separate PR. 
I am currently working on my own certified-assets-based chunk certification solution. If I succeed, then I will definitely share them in the PR format in order to at least discuss the solution

[the remark this answers is not in the document]
Comment (jplevyak) [reply src/ic-certified-assets/src/lib.rs:207]: SGTM

[the remark this answers is not in the document]
Comment (3cL1p5e7) [reply src/ic-certified-assets/src/lib.rs:213]: @jplevyak done :)
https://github.com/dfinity/cdk-rs/pull/219

**Pull request closed without merging**